.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+**Description of the change**
+
+Clearly and concisely describe the purpose of the pull request. If this PR relates to an existing issue or change proposal, please link to it. Include any other background context that would help reviewers understand the motivation for this PR.
+
+---
+
+**Checklist:**
+
+- [ ] Added the change to the changelog's "Unreleased" section with a reference to this PR (e.g. "- Made a change (#22)")
```

Comment:
I'm not 100% sure about this, but we may need to use backticks instead of quotes, or else opening a PR may actually attempt to link to a PR or issue numbered 22.

Reply:
What about using the non-existent issue #000 ?

Reply:
So long as it doesn’t end up generating a link that’s fine with me!

Reply:
I've updated the PR to use that non-existent issue. Good catch!